### DISCUSSION TITLE
Expected Failure type

### DISCUSSION THREAD

**Discussion by alexnot95** [lyndsey-ferguson/fastlane-plugin-test_center#359] [General]
Hi @lyndsey-ferguson, I have a question about new type in xcresult report, as I know sometime ago Apple added new method - https://developer.apple.com/documentation/xctest/3726077-xctexpectfailure so If we will use this method in our tests, we can see types: Passed, Failed, Skipped and Expected Failure
<img width="443" alt="Screenshot 2021-10-19 at 17 39 49" src="https://user-images.githubusercontent.com/37154209/137933271-2f788513-652d-4828-829c-732efcdd16ee.png">
p?

Is it possible to add :expected_failure type to tests_from_xcresult action? Thanks

**Comment by lyndsey-ferguson** [comment]
Hello @alexnot95 I don't think I'd be adding support for that status right away, but I am open to PRs. Here is where you would make the change: https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/lib/fastlane/plugin/test_center/actions/tests_from_xcresult.rb#L42

**Reply by alexnot95** [reply]
@lyndsey-ferguson ok, created a PR https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/pull/360
